wagtail/admin/templates/wagtailadmin/pages/action_menu/menu.html

```diff
@@ -1,6 +1,8 @@
 {% if show_menu %}
     <div class="dropdown-toggle icon icon-arrow-up"></div>
-    <ul role="menu">
-        {% for item in rendered_menu_items %}{{ item }}{% endfor %}
-    </ul>
+    <nav aria-label="Page actions">
```

Comment:
We'll need this to be:

```suggestion
    <nav aria-label="{% trans 'Page actions' %}" role="navigation">
```

so it's translateable.

I also think we should keep adding `role="navigation"` like for existing `nav` landmarks – I’ve understood that this would help with support across a wider range of assistive technologies, although I'm not entirely sure which support what.

Reply:
After our research and your testing, I'll leave adding `role="navigation"`, but I will add in the `{% trans %}` tag, good spot.